Content.Server/Chat/ChatCommands.cs

```diff
@@ -26,10 +28,17 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
 
             var message = string.Join(" ", args);
 
+
             if (player.AttachedEntity.HasComponent<GhostComponent>())
                 chat.SendDeadChat(player, message);
             else
-                chat.EntitySay(player.AttachedEntity, message);
+            {
+                // This is a hacky way of getting the Players Mind
+                var mindComponent = player.ContentData().Mind;
```

Comment:
This could probably be tidied up a bit if there was getters/setters for the player object